### ISSUE TITLE
Chat bot auto train english?

### ISSUE THREAD

**Issue by viet20062000** [gunthercox/ChatterBot#2137]
Hi, I'm newbie in this field also with chatterbot, I want to ask about the trainer, I'm implementing a chat bot in Vietnamese which is not supported currently, so I've to create the data by my own. It took me a while but generally I've done it. But when I choose trainers to train with this dataset, the return value still english and when I export the chat bot it's still english. Can someone help me?

**Comment by farooqkz** [comment]
Are you using chatterbot corpus for training your bot?
Because that doesn't support your language as far as I see: https://github.com/gunthercox/chatterbot-corpus/tree/master/chatterbot_corpus/data

**Comment by viet20062000** [comment]
> Are you using chatterbot corpus for training your bot?
> Because that doesn't support your language as far as I see: https://github.com/gunthercox/chatterbot-corpus/tree/master/chatterbot_corpus/data
yes I'm using chatterbot corpus, as my knowledge about chatterbot, it's using spacy for NLP, so I've installed a pipeline for my language and develop the dataset for my language, add it to the link you posted. Generally I've done it but even though I didn't assign my bot to train english, it's still automatically train it (because when I export the bot data, it's still have English data). i don't know why.

**Comment by viet20062000** [comment]
yes I'm using chatterbot corpus, as my knowledge about chatterbot, it's using spacy for NLP, so I've installed a pipeline for my language and develop the dataset for my language, add it to the link you posted. Generally I've done it but even though I didn't assign my bot to train english, it's still automatically train it (because when I export the bot data, it's still have English data). i don't know why.

**Comment by farooqkz** [comment]
What do you mean by "add it to the link you posted"? opened a PR?
and by the way, are you using the same database for both English trained bot and the bot trained in your language?
I mean training data are saved in database as far as I know.
